local-cli/bundle.js

```diff
@@ -35,7 +35,7 @@ function getBundle(flags) {
     projectRoots: projectRoots,
     transformModulePath: require.resolve('../packager/transformer.js'),
     assetRoots: assetRoots,
-    cacheVersion: '2',
+    cacheVersion: '3',
```

Comment:
This file is not synced internally. Can you submit a different pull request for this one?